demos/wasi/simpleserver/Cargo.toml

```diff
@@ -0,0 +1,9 @@
+[package]
+name = "simpleserver"
+version = "0.1.0"
+authors = ["Matt Butcher <matt.butcher@microsoft.com>"]
+edition = "2018"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
```

Comment:
Can you add an empty `[workspace]` block here? It silences a warning from the top level cargo that it has some unknown thing in its workspace

Reply:
Oh, _that's_ how you do it? Google had utterly failed me